src/Common/PlatformLinux.inl

```diff
@@ -1100,6 +1100,11 @@ inline void restore_path_separators(char * path)
     while (char* sep = strchr(path, '/')) *sep = '\\'; //
 }
 
+inline void platform_path_separators(char * path)
```

Comment:
This function is a duplicate of `convert_path_separators` function 🙃

Reply:
Fixed